odoo-shippable/scripts/build-image.sh

```diff
@@ -129,6 +129,18 @@ wget -q -O /usr/share/vim/vim74/spell/es.utf-8.spl http://ftp.vim.org/pub/vim/ru
 sed -i 's/ set mouse\=a/\"set mouse\=a/g' ~/.vimrc
 sed -i "s/let g:neocomplete#enable_at_startup = 1/let g:neocomplete#enable_at_startup = 0/g" ~/.vimrc
 
+# Install YouCompleteMe
+apt-get install cmake
```

Comment:
@JesusZapata IMHO you can add the repo [here](https://github.com/vauxoo-dev/docker-odoo-image/blob/8c19c4dd7ab28c78b712bb71c372753dc6f4bf58/odoo-shippable/scripts/build-image.sh#L59) so it will be updated [here](https://github.com/vauxoo-dev/docker-odoo-image/blob/8c19c4dd7ab28c78b712bb71c372753dc6f4bf58/odoo-shippable/scripts/build-image.sh#L62) in a single update (this is a little fast), besides you can append the needed packages to the [list](https://github.com/vauxoo-dev/docker-odoo-image/blob/8c19c4dd7ab28c78b712bb71c372753dc6f4bf58/odoo-shippable/scripts/build-image.sh#L42-L47).

With the previous you'll have in [this block](https://github.com/vauxoo-dev/docker-odoo-image/blob/8c19c4dd7ab28c78b712bb71c372753dc6f4bf58/odoo-shippable/scripts/build-image.sh#L135-L144) the YouCompleteMe installation only